install_config/aggregate_logging.adoc

```diff
@@ -1370,28 +1439,42 @@ Curator. Currently the `logging-curator` ConfigMap is used to configure both
 your ops and non-ops Curator instances. Any `.operations` configurations are
 in the same location as your application logs configurations.
 
-. To edit the provided ConfigMap to configure your Curator instances:
+. To create the Curator configuration, edit the configuration in the deployed
+ConfigMap:
 +
 ----
 $ oc edit configmap/logging-curator
 ----
 
-. To replace the provided ConfigMap instead:
+. For scripted deployments, copy the configuration file that was created by the
+installer and create your new OpenShift custom config:
 +
 ----
-$ create /path/to/mycuratorconfig.yaml
-$ oc create configmap logging-curator -o yaml \
-  --from-file=config.yaml=/path/to/mycuratorconfig.yaml | \
-  oc replace -f -
+$ oc extract configmap/logging-curator --keys=curator5.yaml,config.yaml --to=/my/config
+  edit /my/config/curator5.yaml
+  edit /my/config/config.yaml
+$ oc delete configmap logging-curator ; sleep 1
+$ oc create configmap logging-curator \
+    --from-file=curator5.yaml=/my/config/curator5.yaml \
+    --from-file=config.yaml=/my/config/config.yaml \
+    ; sleep 1
 ----
 
-. After you make your changes, redeploy Curator:
+. Or if you are using the *actions file*:
```

Comment:
If this is a possible replacement for the previous step, then I'd turn both of them into substeps of a common ancestor step.